mix.exs

```diff
@@ -22,8 +22,7 @@ defmodule NewRelic.Mixfile do
 
   def application do
     [
-      extra_applications: [:logger, :inets, :ssl],
-      included_applications: [:os_mon],
+      extra_applications: [:logger, :inets, :ssl, :os_mon],
```

Comment:
I put it in `included_applications` because `extra_applications` are always started, and I wanted to prevent `os_mon` from starting unless the agent is actually enabled (https://github.com/newrelic/elixir_agent/pull/473).

I didn't see that there's a way to specify them as optional however, which is probably what we want:

```suggestion
      extra_applications: [:logger, :inets, :ssl, {:os_mon, :optional}],
```

https://hexdocs.pm/mix/1.18.4/Mix.Tasks.Compile.App.html